contract-tests/tests/test/amazon/misc/service_name_in_env_var_test.py

```diff
@@ -0,0 +1,20 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+from typing import List
+
+from resource_attributes_test_base import ResourceAttributesTest, _get_k8s_attributes
+from typing_extensions import override
+
+
+class ServiceNameInResourceAttributesTest(ResourceAttributesTest):
+
+    @override
+    # pylint: disable=no-self-use
+    def get_application_otel_resource_attributes(self) -> str:
+        pairlist: List[str] = []
+        for key, value in _get_k8s_attributes().items():
+            pairlist.append(key + "=" + value)
+        return ",".join(pairlist)
```

Comment:
Not blocking: Advise putting this into a helper method in ResourceAttributesTest, like `toResourceAttributesEnvVar`.

Reply:
After thinking about it, believe keep it as-is will show reader more clear picture for what's the difference between three test class. suggest keep it as-is